includes/services/OpenLayers/jquery.openlayers.js

```diff
@@ -100,6 +100,7 @@
 				markerData.lonlat.transform(new OpenLayers.Projection("EPSG:4326"), new OpenLayers.Projection("EPSG:900913"));
 			}
 
+			if (this.bounds != null) this.bounds.extend(markerData.lonlat); // Extend the bounds when no center is set.
```

Comment:
Does this mean that once the map has been fully loaded, and some event is causing addMarker to be called, the map will re-zoom and re-center?

(I don't really recall how the bounds stuff works, and if this rezoom and re-center happens if something gets added to it or if you need to make another call somewhere)